src/Core/Usecase/Form/VerifyFormLoaded.php

```diff
@@ -45,7 +45,10 @@ protected function verifyFormExists()
 	// Usecase
 	public function interact()
 	{
-		$this->verifyFormExists();
+		if ($this->getRequiredIdentifier('form_id')) {
```

Comment:
Doesn't `getRequiredIdentifier` throw an error if `form_id` hasn't been set?
I think you just need `getIdentifier` ?